src/steps/authenticate.js

```diff
@@ -77,20 +101,15 @@ export async function authenticate(state, req, res) {
 
   // validate jti
   if (jti) {
-    const ids = Array.isArray(state.config.access.apiKeyId)
-      ? state.config.access.apiKeyId
-      : [state.config.access.apiKeyId];
-    if (ids.indexOf(jti) < 0) {
-      state.log.warn(`[auth] invalid jti ${jti}: does not match configured id ${state.config.access.apiKeyId}`);
+    if (access.apiKeyId.indexOf(jti) < 0) {
```

Comment:
I see, `apiKeyId` is an array, first looked like a "string contains" check. I guess the property name `apiKeyId` must be kept and couldn't be named `apiKeyIds`, right?

Reply:
yes. most of the properties that can be configured are either strings or string arrays. we could of course use `apiKeyIds` and `allows` in the structure returned from `getAccessConfig()` but then it would be intransparent.